src/database/models/tripRequest.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const tripRequest = sequelize.define('tripRequest', {
+    userId: DataTypes.INTEGER,
+    originId: DataTypes.INTEGER,
+    destinationId: DataTypes.INTEGER,
+    reason: DataTypes.STRING,
+    tripTypeId: DataTypes.INTEGER,
+    statusId: DataTypes.INTEGER,
+    startDate: DataTypes.STRING,
+    returnDate: DataTypes.STRING
+  }, {});
+  tripRequest.associate = function(models) {
+    tripRequest.belongsTo(models.users, {
+      as: 'user',
+      targetKey: 'id',
+      sourceKey: 'userId',
+    })
```

Comment:
Remember to add in all migrations and models association `ON CASCADE DELETE`. if a row of the referenced table is deleted, then all matching rows in the referencing table are deleted.